homeassistant/components/generic_thermostat/config_flow.py

```diff
@@ -69,6 +79,30 @@
             mode=selector.NumberSelectorMode.BOX, unit_of_measurement=DEGREE, step=0.1
         )
     ),
+    vol.Optional(CONF_PRECISION): selector.SelectSelector(
```

Comment:
This is quite right yet, because it saves the config value as a string (`"0.1"`) instead of a float (`0.1`), and I can't work out how to clean that up. 😞